README.md

```diff
@@ -13,6 +13,7 @@ pip install tflearn
 pip install nltk
 pip install beautifulsoup4
 pip install lxml
+pip install --upgrade gensim
```

Comment:
it can just be this if people haven't already installed
```suggestion
pip install gensim
```

Reply:
`pip install gensim` didn't work